Gemfile

```diff
@@ -148,6 +148,7 @@ group :development do
 end
 
 group :test, :development do
+  gem 'byebug'
```

Comment:
I use this gem for debugging

Reply:
So did I, but the latest and recommended debugging gem is 'debug'.